tests/src/common/WhiskProperties.java

```diff
@@ -235,6 +235,10 @@ public static int getControllerPort() {
         return Integer.parseInt(whiskProperties.getProperty("controller.host.port"));
     }
 
+    public static int getMaxActionInvokesPerMinute() {
+        return Integer.parseInt(whiskProperties.getProperty("limits.actions.invokes.perMinute"));
```

Comment:
@mdeuser could address concern about required key, and optional key.
The key `limits.actions.invokes.perMinute` might not be present, it's optional

```
val maximumInvokesPerMinute = getLimit("defaultLimits.actions.invokes.perMinute", "limits.actions.invokes.perMinute")


    println(s"maximumInvokesPerMinute  = $maximumInvokesPerMinute")

    /*
     * Retrieve a numeric limit for the key from the property set.  If the overrideKey is present, use that.
     */
    def getLimit(key: String, overrideKey: String) = Try {
        WhiskProperties.getProperty(overrideKey).toInt
    } getOrElse {
        WhiskProperties.getProperty(key).toInt
    }
```